chatwoot/chatwoot
# chore: Fix emails being sent with the wrong translations

Fix for emails being sent with the wrong translations

## QA Instructions 
------------------

This PR introduces config changes to enable account creation and switching accounts from the dashboard for the default test account.  This could be used to test the changes easily in local environments

<img width="818" alt="Screenshot 2021-05-07 at 4 19 39 PM" src="https://user-images.githubusercontent.com/73185/117439124-120cb800-af50-11eb-8f80-ba2d72e95481.png">


- Create multiple accounts for a user through the super admin panel or via the dashboard 
- Assign different languages to each of these accounts 
- Switch over into various accounts via  dashboard account switcher and trigger cases where emails are sent out 
- Ensure each email is sent according to the correct account language 

notes:  only some of the chatwoot emails are translated
- titles of auth related emails ( account confirmation. password reset, agent confirmation )
- titles of conversation continuity and transcript emails
